conf/application.conf

```diff
@@ -180,6 +180,7 @@ feature-switch {
   enable-income-sources-new-journey = false
   enable-time-machine = false
   enable-session-data-storage = true
+  enable-review-and-reconcile-debits = true
```

Comment:
in confluence document, you set this to false for some reason;
if we can synch these pls => https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?spaceKey=MISUV&title=Feature+Switches+State